streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -725,8 +725,10 @@ private boolean assignTasksToClients(final Set<String> allSourceTopics,
                                                                    statefulTasks,
                                                                    assignmentConfigs);
 
-        log.info("Assigned tasks to clients as {}{}.",
-            Utils.NL, clientStates.entrySet().stream().map(Map.Entry::toString).collect(Collectors.joining(Utils.NL)));
+        log.info("Assigned tasks to clients as: {}{}.", Utils.NL,
```

Comment:
This is unnecessarily verbose, plus part of it is replaced by line 960 below, so I trimmed a bit here.